kuma/landing/templates/landing/newsfeed.html

```diff
@@ -5,10 +5,10 @@
       <h2 class="entry-title"><a href="{{ e.link }}" rel="bookmark">{{ e.title|entity_decode }}</a></h2>
       <p class="entry-summary">{{ e.summary|strip_tags|entity_decode|truncatewords(50) }}</p>
       <p class="entry-meta vcard">
-      {% trans authorlink='<cite class="author fn">{0}</cite>'|f(e.author)|safe,
-               entrydate='<time class="updated" datetime="{0}" title="{0}">{1}</time>'|f(
-                 entry.last_published|datetime('%Y-%m-%d'), entry.last_published|datetime)|safe %}
-      Posted {{ entrydate }} by {{ authorlink }}
+      {% set author_link = '<cite class="author fn">%s</cite>' % e.author %}
+      {% set entry_date = '<time class="updated" datetime="%(stamp)s" title="%(stamp)s">%(date)s</time>' % {'stamp': entry.last_published|datetime('%Y-%m-%d'), 'date': entry.last_published|datetime} %}
```

Comment:
I'd prefer the block version of the set tag here instead since it's more readable.